### ISSUE TITLE
Add yarn version to dev set up docs

### ISSUE THREAD

**Issue by cquinn540** [activist-org/activist#998]
### Terms

- [X] I have searched [open and closed feature requests](https://github.com/activist-org/activist/issues?q=is%3Aissue+label%3Afeature)
- [X] I agree to follow activist's [Code of Conduct](https://github.com/activist-org/activist/blob/main/.github/CODE_OF_CONDUCT.md)

### Description

I think it would be helpful for new developers to mention which version of yarn this project uses in the docs since yarn 2 and beyond are out.

I'm assuming as of now it's yarn 1.22.22 from running `docker exec nuxt_frontend yarn --version` but that's probably not a beginner friendly solution

Something like:

<details><summary><strong>Frontend: Yarn</strong></summary>
<p>

The frontend currently uses [yarn 1](https://classic.yarnpkg.com/en/docs/install).

```bash
# In the root activist directory:
cd frontend
yarn install
yarn run dev
```

### Contribution

I can open a pull request for this

**Comment by cquinn540** [comment]
This should be labelled 'documentation', but I don't think I have the permissions to do that?

**Comment by cquinn540** [comment]
I think upgrading yarn makes sense.  I haven't done that kind of migration before, so I am not sure how much work it will be

**Comment by andrewtavis** [comment]
Thing is that for Homebrew 1.22.22 is the most up to date version, so maybe for now just send along the docs update 😊 We can do that upgrade in a new issue later :)

**Comment by cquinn540** [comment]
Looking into it I agree that actually upgrading should be a separate issue.  There are breaking changes going from yarn 1 to yarn 2+ so it may not be trivial.

**Comment by andrewtavis** [comment]
Ok :) Feel free to send along the docs PR and we'll go from there!

**Comment by andrewtavis** [comment]
Closed by #999 🚀 One more to 1,000 :) Thanks for this, @cquinn540!
